assets/component-swatch-input.css

```diff
@@ -1,15 +1,27 @@
 /* swatch-input lives in its own file for reusability of the swatch in other areas than the product form context */
+
 .swatch-input__input + .swatch-input__label {
   --swatch-input--size: 4.4rem;
   --swatch-input--border-radius: 50%;
 
   display: inline-block;
   border-radius: var(--swatch-input--border-radius);
   cursor: pointer;
-  outline-offset: 0.2rem;
+  outline-offset: -0.2rem;
   forced-color-adjust: none;
 }
 
+  .swatch-input__input + .swatch-input__label {
+  --swatch-input--size: 3.6rem;
+  font-size: var(--swatch-input--size);
+  width: 44px;
+  height: 44px;
+  display: flex;
+  justify-content: center;
+  align-items: center
+
+}
+
```

Comment:
Here are a couple funky things I noticed: 
<details>
<summary>Desktop looks a bit off now</summary>

![](https://screenshot.click/08-00-lvpde-51n4c.png)
</details>

<details>
<summary>Here is how it looks when receiving focus</summary>

![](https://screenshot.click/08-01-mjw1m-sfb3j.png)
</details>

Reply:
The focus rings are going to be a really tricky part of this.   I don't suggest trying to include this in the next release, personally.